### PR TITLE
tests, multus: Rename NAD format string constants

### PR DESCRIPTION
**Pull request by orelmisan**
**What this PR does / why we need it**:
Improve test readability by renaming and reordering the
NAD format string constants

Signed-off-by: Orel Misan <omisan@redhat.com>

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/network/vmi_multus.go [hunk 1]
```diff
@@ -61,12 +61,12 @@ import (
...
+	linuxBridgeConfNAD     = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"mynet\", \"plugins\": [{\"type\": \"%s\", \"bridge\": \"%s\", \"vlan\": %d, \"ipam\": {%s}, \"macspoofchk\": %t, \"mtu\": 1400},{\"type\": \"tuning\"}]}"}}`
+	ptpConfNAD             = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"mynet\", \"plugins\": [{\"type\": \"ptp\", \"ipam\": { \"type\": \"host-local\", \"subnet\": \"%s\" }},{\"type\": \"tuning\"}]}"}}`
+	macvtapNetworkConfNAD  = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
```
Comment (EdDev) [inline tests/network/vmi_multus.go:66]: You seem to have changed the order and now it is hard to review. Can you please leave the order intact for the renaming commit?
If you still want to re-order these, just to it in another PR.

**Review by EdDev — COMMENTED** [review]
Looks fine to me, just the reordering messes up the diff a bit. Try to see if you can make it easier to review it. See inline comment.

File: tests/network/vmi_multus.go (hunk 1 above)
Comment (orelmisan) [reply tests/network/vmi_multus.go:66]: Done

**Review by EdDev — APPROVED** [review]
Thanks

**Pull request merged**